src/CognitiveServices/SpeechSynthesis.ts

```diff
@@ -143,6 +143,67 @@ class CognitiveServicesHelper {
     private readonly _tokenURL = "https://api.cognitive.microsoft.com/sts/v1.0/issueToken";
     private readonly _synthesisURL = "https://speech.platform.bing.com/synthesize";
     private readonly _outputFormat = "riff-16khz-16bit-mono-pcm";
+    private readonly _femaleVoiceMap = new Map([
```

Comment:
Can you put the maps at the bottom of the file? This way someone doesn't have to scroll through them all to get to the code.